src/BuildServer/Services/ArtifactBuilders/ArtifactBuilder.cs

```diff
@@ -59,5 +64,19 @@ public bool Build(Build build)
                 process.Dispose();
             }
         }
+
+        private string ValidateParameter(string parameter)
+        {
+            Regex regex = new Regex(@"^[a-zA-Z0-9./\-_]*$");
+            Match match = regex.Match(parameter);
+            if (match.Success)
+            {
+                return parameter;
+            }
+
+            this.logger.LogError($"Invalid parameter. " +
+                $"Only alpha-numeric, '/', '.', '-', '_' characters are allowed");
+            return null;
```

Comment:
nit: should we throw an exception here instead of returning `null`? Is there a scenario where we'd want to continue with a command different from what the user intended instead of terminating early?

Reply:
We might change the way we are handling errors in the future, for now bash's error message will suffice